### PR TITLE
bump chadrc_types.lua

### PR DESCRIPTION
**Pull request by mgastonportillo**
'chadracula-evondev' was missing in the ThemeName type, triggering diagnostics.

Let me know if I overlooked anything.

### CODE REVIEW

**Comment by lucario387** [comment]
Run the script.

**Comment by mgastonportillo** [comment]
Got it, should I squash and force push or send the PR again?

**Comment by lucario387** [comment]
yes, please

**Comment by mgastonportillo** [comment]
Fixed a mistake in the commit message. Now it should address the changes properly.

File: nvchad_types/themes.lua [hunk 1]
```diff
@@ -34,18 +37,26 @@ vim.g.nvchad_theme = 'onedark'
...
+---| '"jabuti"'
 ---| '"gruvchad"'
 ---| '"gruvbox_light"'
 ---| '"gruvbox"'
 ---| '"github_light"'
 ---| '"github_dark"'
 ---| '"gatekeeper"'
+---| '"flexoki"'
+---| '"flexoki-light"'
+---| '"flex-light"'
```
Comment (lucario387) [inline nvchad_types/themes.lua:59]: There is a lot more themes then?

Comment (mgastonportillo) [reply nvchad_types/themes.lua:59]: I think so. I didn't add anything manually other than the initial changes, but that's been overwritten by the script output.

**Review by lucario387 — APPROVED** [review]

**Pull request merged**